### PR TITLE
BDCS-1998 Scheduled payment notice

### PR DESCRIPTION
**Pull request by sean-hill**

### CODE REVIEW

File: source/includes/events/_bill.scheduled-payment-notice.md.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            "current": "2019-03-28 17:00:00.000"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Friendly notice for when the due date changes after a bill payment has been scheduled.
```
Comment (kolinkodm) [inline source/includes/events/_bill.scheduled-payment-notice.md.erb]: When the due date changes after a bill payment has been scheduled.

Comment (kolinkodm) [reply source/includes/events/_bill.scheduled-payment-notice.md.erb]: "Friendly" is too subjective, I think -- and also following the format of the current event types, the description starts with "When ___"

File: source/includes/events/_bill.scheduled-payment-notice.md.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Attribute | Description
+--------- | -----------
+`type` | `bill.scheduled-payment-notice`
+`payload.data.company` | The [company object](#the-company-object) linked to this resource.
+`payload.data.bill` | Object containing the names of the bill attributes that have changed and their previous values if available.
```
Comment (kolinkodm) [inline source/includes/events/_bill.scheduled-payment-notice.md.erb]: The bill object containing attributes that have changed and their current and previous values, if available.

- I added a comma before if
- Based on the example in the docs, from what I can see the payload.data.bill returns the current and previous attributes (previous if available) from the bill object, so I rewrote it to better reflect that. I also rewrote to follow the same format as the payload.data.company above it -- which I would recommend linking "bill object" to the Bill object in the docs, like the company object.

**Comment by sean-hill** [comment]
image

**Pull request merged**